docs/reference/env-variables.rst

```diff
@@ -0,0 +1,657 @@
+.. meta::
+    :description: Environment variables reference
+    :keywords: AMD, ROCm, Environment variables, Environment, reference
+
+.. _env-variables-reference:
+
+*************************************************************
+Environment variables reference
+*************************************************************
+
+General overview
+==========================================
+
+Environment variables effecting the all the libraries in ROCm software stack
```

Comment:
Typo - remove 'the'

Environment variables effecting **the** all the libraries in ROCm software stack.

Do we know that these variables impact 'all' the libraries? We could consider,

Environment variables that impact libraries in the ROCm software stack.

Reply:
'Affect' has a negative connotation, so not recommended.

Reply:
Updated.